File: frontend/src/app/pages/models/tableMasterDto.ts

export interface TableMasterDto {
    date: Date;
    showRadioButton: number;
    quantity : number;
    number4 : number; 
    number5 : number; 
    number6 : number; 
    codeProduct: string;
    codeOwner: string;
    codeSupplier: string;
    idLocation : number;
    idInventoryStatus: number;

}
export interface SaveDoubleForm {
    idProduct: number;
    idOwner: number;
    idSupplier: number;
    idLocation: number;
    idInventoryStatus: number;
    showRadioButton: number;
    quantity : number;
    number4 : number; 
    number5 : number; 
    number6 : number;
    date: Date;
    idFormSetSeparate: number;
    totalTong: number;
  

}

// export class saveDoubleFormClass implements SaveDoubleForm{
//     public idProduct!: number;
//     public idOwner!: number;
//     public idSupplier!: number;
//     public idLocation!: number;
//     public idInventoryStatus!: number;
//     public showRadioButton!: number;
//     public quantity! : number;
//     public number4! : number; 
//     public number5! : number; 
//     public number6! : number;
//     public date!: Date; 
// } 

File: backend/train-vimash-back-end/src/main/java/tech/getarrays/employeemanager1/dto/TableMasterDTO.java
package tech.getarrays.employeemanager1.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.beans.factory.annotation.Autowired;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class TableMasterDTO {

    private Date date;

    private Integer showRadioButton;

    private Integer quantity;

    private Integer number4;

    private Integer number5;

    private Integer number6;

    private String codeProduct;

    private String codeOwner;

    private String codeSupplier;

    private Integer idLocation;

    private Integer idInventoryStatus;



}
